pkgs/applications/window-managers/sway/wrapper.nix

```diff
@@ -23,6 +24,7 @@ let
        ${extraSessionCommands}
        export _SWAY_WRAPPER_ALREADY_EXECUTED=1
      fi
+   '' + lib.optionalString dbusSupport ''
```

Comment:
won't this mean that if dbus support is not enabled then the baseWrapper won't have a `exec ${sway}/bin/sway "$@"`

Reply:
Yikes, thanks for catching this.  I don't use the wrapper script which is why I didn't notice.  Sorry about that; will push another commit to fix this.

Reply:
Okay, pushed.

I tried to write the conditional logic in the most maintainable way I could.  The doubly-nested antiquotation isn't pretty, but wrapping the entire `if`..`else`..`fi` block in `lib.optional` and then adding a completely separate code path seemed even worse.